src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -137,8 +137,8 @@ public void SimpleBuild()
             _logger.AssertLogContains("[success]");
             Assert.Single(_logger.ProjectStartedEvents);
 
-            ProjectStartedEventArgs projectStartedEvent = _logger.ProjectStartedEvents[0];
```

Comment:
nit: Do we have any tests that checks the properties and items in ProjectStartedEventArgs? Given that we still support them being there for some cases, we should have some test for it.

Reply:
Yes - e.g. `OutOfProcNodeForwardCertainproperties` or the tests that explicitly set `MSBUILDLOGPROPERTIESANDITEMSAFTEREVALUATION` to '0'